### ISSUE TITLE
[BUG] Missing sections in documentation

### ISSUE THREAD

**Issue by rubencalje** [pastas/pastas#705]
**Describe the bug**
I am missing some sections on the current documentation website. For example in version 0.18 there was a notebook about the Hantush-resonse-functions: https://pastas.readthedocs.io/en/v0.18.0/concepts/hantush_response.ipynb.html. I cannot find this section in the current documentation: https://pastas.readthedocs.io/en/latest/index.html. Did we remove this notebook on purpose? Or should it be placed in "benchmarks"?

**Comment by raoulcollenteur** [comment]
Yes we did that on purpose. That notebook was merged with another one about response functions and added here: https://pastas.readthedocs.io/en/latest/benchmarks/check_response_functions.html

**Comment by rubencalje** [comment]
ok, great. There are some sections in the original notebook that are missing from https://pastas.readthedocs.io/en/latest/benchmarks/check_response_functions.html or https://pastas.readthedocs.io/en/latest/examples/multiple_wells.html. For example the section "Which Hantush should I use?".

But mainly I am missing the synthetic example, which shows you get the (almost exact) same var_gain and std_gain with HantushWellModel and WellModel, which is important. Therefore, I am in favor of adding this section to the benchmarks. I can do that.

**Comment by raoulcollenteur** [comment]
Was there any progress lately on this or was it included in another notebook?
